cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -101,43 +98,23 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run existing copyrigh
     private fun configureScanner(scannerConfiguration: ScannerConfiguration?): Scanner {
         val config = scannerConfiguration ?: ScannerConfiguration()
 
-        // By default use a file based scan results storage.
-        val localFileStorage = XZCompressedLocalFileStorage(getUserOrtDirectory().resolve("$TOOL_NAME/results"))
-        val fileBasedStorage = FileBasedStorage(localFileStorage)
-        ScanResultsStorage.storage = fileBasedStorage
-
-        // Allow to override the default scan results storage.
-        val configuredStorages = listOfNotNull(
-            config.fileBasedStorage,
-            config.postgresStorage
-        )
-
-        require(configuredStorages.size <= 1) {
-            "Only one scan results storage may be configured."
-        }
-
-        configuredStorages.forEach { ScanResultsStorage.configure(it) }
+        ScanResultsStorage.configure(config)
 
         val scanner = scannerFactory.create(config)
 
         println("Using scanner '${scanner.scannerName}' with storage '${ScanResultsStorage.storage.name}'.")
 
-        val localFileStorageLogFunction: ((String) -> Unit)? = when {
-            // If the local file storage is in use, log about it already at info level.
-            log.delegate.isInfoEnabled && ScanResultsStorage.storage == fileBasedStorage -> log::info
-
-            // Otherwise log about the local file storage only at debug level.
-            log.delegate.isDebugEnabled -> log::debug
+        ScanResultsStorage.storage.let { storage ->
+            if (storage is FileBasedStorage) {
+                val backend = storage.backend
```

Comment:
Nit: Here you are using a `val`, two lines above a `let` to assign a temporary variable. It would seem more consistent to also use a `val` above, which would also reduce the nesting level.

Reply:
Done.